### PR TITLE
Adjust docfx build for devsite (old - keeping temporarily for posterity)

### PR DESCRIPTION
**Pull request by jskeet**
This is only a draft as there's a FIXME about api_shortname

cc @tbpg

### CODE REVIEW

File: uploaddocs.sh [hunk 1]
```diff
@@ -66,12 +66,14 @@ do
     echo "Final upload stage (googleapis.dev)"
     python -m docuploader upload . --credentials $SERVICE_ACCOUNT_JSON --staging-bucket $GOOGLEAPIS_DEV_STAGING_BUCKET
 
-    cd ../devsite/api
+    # For DevSite, we just upload the yaml in the obj/api directory
+    cd ../obj/api
     echo "Generating metadata (DevSite)"
     python -m docuploader create-metadata --name $pkg --version $version --language dotnet --github-repository googleapis/google-cloud-dotnet
```
Comment (tbpg) [inline uploaddocs.sh:72]:
```suggestion
    # FIXME: WE DON'T CURRENTLY KNOW API_SHORTNAME
    api_shortname=todo
    python -m docuploader create-metadata --name $pkg --version $version --language dotnet --github-repository googleapis/google-cloud-dotnet --serving-path /dotnet/docs/reference/$api_shortname/$pkg/latest
```

File: uploaddocs.sh [hunk 2]
```diff
@@ -66,12 +66,14 @@ do
...
     
     echo "Final upload stage (DevSite)"
-    python -m docuploader upload . --credentials $SERVICE_ACCOUNT_JSON --staging-bucket $DEVSITE_STAGING_BUCKET
+    # FIXME: WE DON'T CURRENTLY KNOW API_SHORTNAME
+    python -m docuploader upload . --credentials $SERVICE_ACCOUNT_JSON --staging-bucket $DEVSITE_STAGING_BUCKET --serving-path /dotnet/docs/reference/$api_shortname/$pkg/latest
```
Comment (tbpg) [inline uploaddocs.sh:76]:
```suggestion
    python -m docuploader upload . --credentials $SERVICE_ACCOUNT_JSON --staging-bucket $DEVSITE_STAGING_BUCKET --destination-prefix docfx
```

**Comment by jskeet** [comment]
Thanks @tbpg - will look at the suggestions tomorrow.

File: uploaddocs.sh (hunk 1 above)
Comment (jskeet) [reply uploaddocs.sh:72]: Ah, my mistake - I'd thought it was in the upload stage for some reason.

File: uploaddocs.sh [hunk 3]
```diff
@@ -66,12 +66,15 @@ do
     echo "Final upload stage (googleapis.dev)"
     python -m docuploader upload . --credentials $SERVICE_ACCOUNT_JSON --staging-bucket $GOOGLEAPIS_DEV_STAGING_BUCKET
 
-    cd ../devsite/api
+    # For DevSite, we just upload the yaml in the obj/api directory
+    cd ../obj/api
     echo "Generating metadata (DevSite)"
-    python -m docuploader create-metadata --name $pkg --version $version --language dotnet --github-repository googleapis/google-cloud-dotnet
+    # FIXME: WE DON'T CURRENTLY KNOW API_SHORTNAME
```
Comment (tbpg) [inline uploaddocs.sh:72]: Looking into this. Not sure the best way to do it yet -- or maybe we should just drop it completely.

**Pull request closed without merging**